add_me_to_your_PATH/Dockerfile.template

```diff
@@ -118,4 +126,4 @@ WORKDIR /JuliaProject
 RUN mkdir -p /root/.julia/config
 RUN echo 'n=8; sleep(n); println("done waiting $n secs for logs/repl_history.jl to sync...")' >> /root/.julia/config/startup.jl
 
-CMD julia --sysimage=/deps.so
+CMD julia
```

Comment:
The final `CMD julia` will not have access to the github token, so users won't be able to add private dependencies?

Reply:
The original code would embed the `GITHUB_TOKEN` into the image which is why it was available for use during runtime. I was planning on punting this functionality to another PR but I've restored it here using K8s secrets to supply the pod with the token.